amd64/container-build.sh

```diff
@@ -0,0 +1,44 @@
+suffix="" # e.g. -hardened
+arch="amd64"
+busybox_version="x86_64"
+dist="http://distfiles.gentoo.org/releases/${arch}/autobuilds/"
```

Comment:
Hope this default URL will always be avaialble and not timeouting. mirrorselect is out of reach at this point :-/